sdk/tests/conformance/textures/misc/compressed-tex-image.html

```diff
@@ -62,6 +62,8 @@
 
   var tex = gl.createTexture();
   gl.bindTexture(gl.TEXTURE_2D, tex);
+  wtu.shouldGenerateGLError(gl, [gl.INVALID_ENUM, gl.INVALID_OPERATION],
```

Comment:
Why is INVALID_OPERATION allowed here?

Reply:
I'm not sure why this was allowed when this test was integrated, but this code doesn't exist any more in the top-of-tree conformance suite.
https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/conformance/textures/misc/compressed-tex-image.html

Even before it was deleted, this had been changed at some point to only allow INVALID_ENUM.
https://github.com/KhronosGroup/WebGL/blob/75650c75640cf3dbda29c8a57b6e4ba7edc4f176/sdk/tests/conformance/textures/misc/compressed-tex-image.html